packages/k2-alpine/src/components/button/Button.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { type FC, type PropsWithChildren } from 'react'
+import { Text, Pressable, useDripsyTheme } from 'dripsy'
```

Comment:
let's create a Primitives.tsx file (similar to the one in the current k2) that contains basic UI components from dripsy. also, let's create a useTheme() that wraps around useDripsyTheme(). you can then use it here or anywhere else that needs it.